### PR TITLE
feat: Add address auto complete

### PR DESCRIPTION
**Pull request by nopparuj917**
- Add address auto complete
- Using the Google Maps Places API

### CODE REVIEW

**Comment by nopparuj917** [comment]
@nnanthicha Can you make the option as cute as Mean's girlfriend (Proud)

File: README.md [hunk 1]
```diff
@@ -18,6 +18,25 @@ You can start editing the page by modifying `pages/index.tsx`. The page auto-upd
...
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+## Package requirement
+- use-places-autocomplete
+
+This package is distributed via [npm](https://www.npmjs.com/package/use-places-autocomplete).
+```sh
+$ yarn add use-places-autocomplete
+# or
+$ npm install --save use-places-autocomplete
+```
```
Comment (noppakorn) [inline README.md:29]: we only use yarn

**Review by noppakorn — CHANGES_REQUESTED** [review]

File: src/pages/account/signup.tsx [hunk 2]
```diff
@@ -157,7 +26,138 @@ type RegistrationFormInput = {
 };
 
 export default function Signup() {
+  const renderForm = (i: number) => {
```
Comment (noppakorn) [inline src/pages/account/signup.tsx:29]: I'm not sure about this but is there any reason for putting this function inside of the Signup function. Can you explain a bit?

File: src/components/places.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  } = usePlacesAutocomplete();
+
+  const onSuggestionClick = (description: string) => {
+    setValue(description, false);
+    clearSuggestions();
+    getGeocode({ address: description }).then((results) => {
+      const { lat, lng } = getLatLng(results[0]);
+      console.log("📍 Coordinates: ", { lat, lng });
+    });
+  };
```
Comment (noppakorn) [inline src/components/places.tsx:22]: Should this be under useCallback?

**Review by noppakorn — CHANGES_REQUESTED** [review]

**Review by noppakorn — APPROVED** [review]

**Pull request merged**